src/test/typescript.ts

```diff
@@ -13,59 +13,106 @@ import { NodePath } from "../node-path";
 import { Visitor } from "../gen/visitor";
 import { Context } from "../path-visitor";
 
-var pkgRootDir = path.resolve(__dirname, "..");
+var pkgRootDir = path.resolve(__dirname, "..", "..");
 var tsTypes = fork([
   esProposalsDef,
   typescriptDef,
   jsxDef,
 ]);
 
-const babelParserDir = path.resolve(__dirname, "data", "babel-parser");
+const babelParserDir = path.resolve(
+  pkgRootDir, "src", "test", "data", "babel-parser");
 
 const babelTSFixturesDir =
   path.join(babelParserDir, "test", "fixtures", "typescript");
 
-glob("**/input.js", {
+glob("**/input.ts", {
   cwd: babelTSFixturesDir,
 }, (error, files) => {
   if (error) {
     throw error;
   }
 
+  if (files.length < 10) {
+    throw new Error(`Unexpectedly few **/input.ts files matched (${
+      files.length
+    }) in ${
+      babelTSFixturesDir
+    }`);
+  }
```

Comment:
Hopefully this check will prevent the scenario where the `glob` pattern suddenly fails to match any files (such as the upstream project deciding to bulk-rename all the `input.ts` files).